homeassistant/components/bang_olufsen/websocket.py

```diff
@@ -92,8 +92,14 @@ def on_notification_notification(
         self, notification: WebsocketNotificationTag
     ) -> None:
         """Send notification dispatch."""
+
         if notification.value:
-            if WebsocketNotification.REMOTE_MENU_CHANGED in notification.value:
+            if WebsocketNotification.BEOLINK.value in notification.value:
```

Comment:
The already existing use of `in` here is indeed quite strange.
I think this should be changed to an equality check in a separate PR:
```diff
         if notification.value:
-            if WebsocketNotification.REMOTE_MENU_CHANGED in notification.value:
+            if WebsocketNotification.REMOTE_MENU_CHANGED == notification.value:
                 async_dispatcher_send(
                     self.hass,
                     f"{self._unique_id}_{WebsocketNotification.REMOTE_MENU_CHANGED}",
                 )
```